cli/slsa-verifier/main_regression_test.go

```diff
@@ -1795,6 +1795,99 @@ func Test_runVerifyNpmPackage(t *testing.T) {
 	}
 }
 
+func Test_runVerifyVSA(t *testing.T) {
+	// We cannot use t.Setenv due to parallelized tests.
+	os.Setenv("SLSA_VERIFIER_EXPERIMENTAL", "1")
+	t.Parallel()
+
+	tests := []struct {
+		name              string
+		AttestationPath   *string
```

Comment:
Can this be `attestationPath` as it doesn't need to be exported?

Reply:
changed